### PR TITLE
Annotated filter.c and lb.c

### PR DESCRIPTION
**Pull request by Pymamid**

### CODE REVIEW

File: projects/suricata-master/human_annotated/filter.c [hunk 1]
```diff
@@ -189,10 +189,10 @@ struct vlan_hdr {
   "call_depth": -1,
   "humanFuncDescription": [
     {
-      "description": "",
-      "author": "",
-      "authorEmail": "",
-      "date": ""
+      "description": "The function extracts the src and dest IP addresses of the IPv4 packet, and checks whether they are in the list of IPs to be dropped. If yes, count of packet drops is incremented by 1, and function returns 0. If not, function returns -1.",
```
Comment (palanik1) [inline projects/suricata-master/human_annotated/filter.c:192]: The function extracts the src and dest IP addresses of the IPv4 packet, and checks whether the dst address is  in the list of IPs to be dropped. If yes, count of packet drops is incremented by 1, and function returns 0. If not, function returns -1."

File: projects/suricata-master/human_annotated/lb.c [hunk 2]
```diff
@@ -217,10 +217,10 @@ static __always_inline int ipv4_hash(struct __sk_buff *skb)
   "call_depth": -1,
   "humanFuncDescription": [
     {
-      "description": "",
-      "author": "",
-      "authorEmail": "",
-      "date": ""
+      "description": "The function takes an IPv6 address and computes a hash by bitwise OR of its four octets.",
```
Comment (palanik1) [inline projects/suricata-master/human_annotated/lb.c:220]: OR or XOR?

**Review by palanik1 — CHANGES_REQUESTED** [review]

File: projects/suricata-master/human_annotated/lb.c (hunk 2 above)
Comment (Pymamid) [reply projects/suricata-master/human_annotated/lb.c]: It should have been XOR. Modified the same.

**Review by palanik1 — APPROVED** [review]

**Comment by palanik1** [comment]
lgtm

**Pull request merged**